tck-flow/src/main/java/org/reactivestreams/tck/FlowSubscriberWhiteboxVerification.java

```diff
@@ -0,0 +1,36 @@
+/************************************************************************
+ * Licensed under Public Domain (CC0)                                    *
+ *                                                                       *
+ * To the extent possible under law, the person who associated CC0 with  *
+ * this code has waived all copyright and related or neighboring         *
+ * rights to this code.                                                  *
+ *                                                                       *
+ * You should have received a copy of the CC0 legalcode along with this  *
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.*
+ ************************************************************************/
+
+package org.reactivestreams.tck;
+
+import org.reactivestreams.Subscriber;
+import org.reactivestreams.tck.flow.support.SubscriberWhiteboxVerificationRules;
+
+/**
+ * Provides tests for verifying {@link org.reactivestreams.Subscriber} and {@link org.reactivestreams.Subscription} specification rules.
+ *
+ * @see org.reactivestreams.Subscriber
+ * @see org.reactivestreams.Subscription
+ */
+public abstract class FlowSubscriberWhiteboxVerification<T> extends SubscriberWhiteboxVerification<T>
+  implements SubscriberWhiteboxVerificationRules {
+
+  protected FlowSubscriberWhiteboxVerification(TestEnvironment env) {
+    super(env);
+  }
+
+  @Override
+  final public Subscriber<T> createSubscriber(WhiteboxSubscriberProbe<T> probe) {
+    return createFlowSubscriber(probe);
+  }
+
+  protected abstract Subscriber<T> createFlowSubscriber(WhiteboxSubscriberProbe<T> probe);
```

Comment:
Shouldn't "Subscriber" here be Flow.Subscriber?

Reply:
Thanks, good catch - have not completed the whitebox (as mentioned, for review here was the "style" how we'll integrate – in the 2 places I put comments). Rest will be done shortly